start/scaling-up/preparing-to-scale-up.md

```diff
@@ -1,92 +1,142 @@
-
-
-### Disk Usage
+of jobs simulateously. Breaking up your work into small, independently executable 
+jobs and optimizing the resource requests of those jobs, by 
+only requesting the amount of memory, disk, and cpus truely needed, 
+will ensure that you get the most out of OSG resources.
+This is an important practice that will reduce the amount of time your 
+jobs remain idle before running and which will maximize your throughput, 
+all helping to get your work completed sooner.
+
+A key aspect to high-throughput computing is to 
+break up your computational work into independently 
+executable tasks whenever possible. Breaking up your work not only 
+increases the parallelization of your work, but also often reduces the 
+memory, disk, and cpus needs for each individual task. If you have questions 
+or are unsure if and how your work can be broken up, please contact us at 
+<support@osgconnect.net>.
+
+This guide will described best pactices and general tips for optimizing 
+your job resource requests **before** scaling up to submitting your full set of jobs. 
+Additional information is also available from the following 2020 OSG Virtual 
+Pilot School lecture video:<a href="https://youtu.be/oMAvxsFJaw4">
+	<img alt="2020 VSP dHTC with HTCondor" src="https://raw.githubusercontent.com/OSGConnect/connectbook/master/images/2020-vsp-intro-dHTC-HTcondor-thumbnail.png" width="360" height="204"></a>
+
+[Introduction to High Throughput Computing with HTCondor](https://youtu.be/oMAvxsFJaw4)
+
+# Always Start With Test Jobs
+
+Submitting test jobs is an important first step for optimizing 
+the resource requests of your jobs. We always recommend submitting a few 
+test jobs first before scaling up whether this is your first time 
+using OSG or you're an experienced user starting a new workflow. 
+Test jobs will also help identify specific `requirements` for your jobs and 
+identify bugs in scripts and workflows.
+
+Some general tips for test jobs:
+
+- Select smaller data sets or subsets of data for your first test jobs. Using 
+smaller data will keep the resource needs of your jobs low which will help get 
+test jobs to start, and complete, sooner.
+
+- If possible, submit test jobs that will reproduce results you've gotten 
+using another system, this makes for a good "sanity check" as you'll be able 
+to compare the results of the test to those previously obtained.
+
+- After initial tests complete successfully, scale up to larger or full-size 
+data sets; if your data spans a range of sizes, submit tests using the smallest 
+and largest data sizes to examine the range of resources that these jobs may need.
+
+- Give your test jobs, and associated HTCondor `log`, `error`, `output`, 
+and `submit` files meaningful names so you know which results refer to which tests.
+
+# Optimize Job Resource Requests
+
+In the HTCondor submit file, you must explicitly request the number of 
+cpus (i.e. cores) and the amount of disk and memory that the job needs 

```

Comment:
I might capitalize CPUs here?

Reply:
I have capitalized CPUs in commit  b69ebd8